docs/user-guides/deployment.md

```diff
@@ -0,0 +1,245 @@
+  discoverer:
+    resources:
+      requests:
+        cpu: 150m
+        memory: 50Mi
+  
+  ## vald-manager settings
+  manager:
+    index:
+      resources:
+        requests:
+          cpu: 150m
+          memory: 30Mi
+      indexer:
+        auto_index_duration_limit: 1m
+        auto_index_check_duration: 40s
+```
+
+</details>
+
+After create `values.yaml`, you can deploy by the following steps.
+
+1. Add vald repo into the helm repo.
+
+    ```bash
+    helm repo add vald https://vdaas.vald.org/charts
+    ```
+
+1. Deploy with the `values.yaml`
+
+    ```bash
+    helm install vald vald/vald --values <YOUR VALUES YAML FILE PATH>
+    ```
+
+### Update Configuration
+
+When you need to update the configuration, you can update by following command with your new `values.yaml`.
+
+```bash
+helm upgrate vald vald/vald --values <YOUR NEW VALUES FILE PATH>
+```
+
+## Using with vald-helm-operator
+
+### Deployment
+
+When deploying with vald-helm-operator, it requires `vr.yaml` file for applying `ValdRelease`.
+`vald-helm-operator` manages the Vald cluster based on configuration of `vr.yaml`.
+
+<details><summary>Sample ValdRelease YAML</summary><br>
+
+```yaml
+apiVersion: vald.vdaas.org/v1
+kind: ValdRelease
+metadata:
+  name: vald-cluster
+# the values of Helm chart for Vald can be placed under the `spec` field.
+spec:
+ defaults:
+   logging:
+     level: debug
+   image:
+     tag: "v1.5.6"
+   server_config:
+     healths:
+       liveness:
+         livenessProbe:
+           initialDelaySeconds: 60
+       readiness:
+         readinessProbe:
+           initialDelaySeconds: 60
+ 
+   ## vald-lb-gateway settings
+   gateway:
+     lb:
+       minReplicas: 2
+       maxReplicas: 2
+   
+   ## vald-agent settings
+   agent:
+     minReplicas: 3
+     maxReplicas: 3
+     podManagementPolicy: Parallel
+     ngt:
+       dimension: 784
+       distance_type: l2
+       object_type: float
+       auto_index_check_duration_limit: 720h
+       auto_index_duration_limit: 24h
+       auto_create_index_pool_size: 10000
+       default_pool_size: 10000
+   
+   ## vald-discoverer settings
+   discoverer:
+     resources:
+       requests:
+         cpu: 150m
+         memory: 50Mi
+   
+   ## vald-manager settings
+   manager:
+     index:
+       resources:
+         requests:
+           cpu: 150m
+           memory: 30Mi
+       indexer:
+         auto_index_duration_limit: 1m
+         auto_index_check_duration: 40s
+```
+
+</details>
+
+After create `vr.yaml`, you can deploy by the following steps.
+
+1. Add vald repo into the helm repo
+
+    ```bash
+    helm repo add vald https://vdaas.vald.org
+     ```
+
+1. Deploy `vald-helm-operator`
+
+    ```bash
+    helm install vald-helm-operator-release vald/vald-helm-operator
+    ```
+
+1. Apply `vr.yaml`
+
+    ```bash
+    kubectl apply -f vr.yaml
+    ```
+
+If you need to auto managing to vald-helm-operator, you must apply `vhor.yaml` for applying `ValdHelmOpearterRelease`.
```

Comment:
I think it may be a little bit hard to let other to understand the relationship of vhor, vr and the cluster 🤔